src/actions/actions.js

```diff
@@ -19,6 +19,7 @@ export const getTeams = () => async dispatch => {
 export const getSelectedTeam = team => async dispatch => {
     dispatch({ type: 'GET_SELECTED_TEAM', payload: team });
     dispatch({ type: 'GET_TEAM_COLOR', payload: team.primaryColor });
+    dispatch({ type: 'PRELOAD_PLAYER', payload: null });
 
     const teamUrlName = (team.teamId === '1610612755' ? 'sixers' : team.ttsName.trim().split(' ').pop().toLowerCase());
```

Comment:
And while we're here, let's change this `team.ttsName.trim().split(' ').pop().toLowerCase()` to
```
team.ttsName.split(' ').splice(-1)[0].toLowerCase()
```

We should try to avoid using `pop()` as it should only be used to manipulate an array.